sktime/tests/_config.py

```diff
@@ -87,8 +87,8 @@
     "STDBSCAN",
     # Temporarily remove RRF from tests, while #7380 is not merged
     "RecursiveReductionForecaster",
-    # DistanceFeatures does ont work for hierarchical data, see #8077
-    "DistanceFeatures",
+    # DistanceFeatures does not work for hierarchical data, see #8077
+    #removed DistanceFeatures
```

Comment:
i think you should just delete the mention of `DistanceFeatures` from these lines